src/DssSpell.t.sol

```diff
@@ -974,48 +1049,257 @@ contract DssSpellTest is DssSpellTestBase {
+
+        // // test L2 side of initBridges
+        assertEq(l2bridge.l1ToL2Token(address(susds)), address(l2susds));
+        assertEq(l2bridge.maxWithdraws(address(l2susds)), type(uint256).max);
+
+        assertEq(l2bridge.l1ToL2Token(address(usds)), address(l2usds));
+        assertEq(l2bridge.maxWithdraws(address(l2usds)), type(uint256).max);
+
+        assertEq(AuthLike(address(l2susds)).wards(address(l2bridge)), 1);
+        assertEq(AuthLike(address(l2usds)).wards(address(l2bridge)), 1);
+
+
+        // ------- Test Deposit -------
+
+        rootDomain.selectFork();
+
+        deal(address(usds), address(this), 100 ether);
+        deal(address(susds), address(this), 100 ether);
+        assertEq(usds.balanceOf(address(this)), 100 ether);
+        assertEq(susds.balanceOf(address(this)), 100 ether);
+
+        usds.approve(address(l1bridge), 100 ether);
+        susds.approve(address(l1bridge), 100 ether);
+        uint256 escrowBeforeUsds = usds.balanceOf(L1_ESCROW);
+        uint256 escrowBeforesUsds = susds.balanceOf(L1_ESCROW);
+
+        l1bridge.bridgeERC20To(
+            address(usds),
+            address(l2usds),
+            address(0xb0b),
+            100 ether,
+            1_000_000,
+            ""
+        );
+        l1bridge.bridgeERC20To(
+            address(susds),
+            address(l2susds),
+            address(0xb0b),
+            100 ether,
+            1_000_000,
+            ""
+        );
+
+        assertEq(usds.balanceOf(address(this)), 0);
+        assertEq(usds.balanceOf(L1_ESCROW), escrowBeforeUsds + 100 ether);
+        assertEq(susds.balanceOf(address(this)), 0);
+        assertEq(susds.balanceOf(L1_ESCROW), escrowBeforesUsds + 100 ether);
+
+        baseDomain.relayFromHost(true);
+
+        assertEq(l2usds.balanceOf(address(0xb0b)), 100 ether);
+        assertEq(l2susds.balanceOf(address(0xb0b)), 100 ether);
+
+       // ------- Test Withdrawal -------
+
+        vm.startPrank(address(0xb0b));
+
+        l2usds.approve(address(l2bridge), 100 ether);
+        l2susds.approve(address(l2bridge), 100 ether);
+
+        l2bridge.bridgeERC20To(
+            address(l2usds),
+            address(usds),
+            address(0xced),
+            100 ether,
+            1_000_000,
+            ""
+        );
+        l2bridge.bridgeERC20To(
+            address(l2susds),
+            address(susds),
+            address(0xced),
+            100 ether,
+            1_000_000,
+            ""
+        );
+        vm.stopPrank();
+
+        assertEq(l2usds.balanceOf(address(0xb0b)), 0);
+        assertEq(l2susds.balanceOf(address(0xb0b)), 0);
+
+        baseDomain.relayToHost(true);
+
+        assertEq(usds.balanceOf(address(0xced)), 100 ether);
+        assertEq(susds.balanceOf(address(0xced)), 100 ether);
     }
 
-    function testNewOsmMomAddition() public {
-        bytes32 ilk = "LSE-MKR-A";
-        address osm = addr.addr("PIP_MKR");
+    function testSparkAllocator() public {
```

Comment:
Wouldn't it be better to turn this into a config test? Maybe creating an extra field in `SystemValues` like `.allocators`?

And again, maybe split this test in 2 parts, one in `t.base.sol` to run every time and the other in `t.sol` with only stuff specific to this spell.

Reply:
This is one time integration same as we have for `checkNewIlkIntegration`, and then we have this Allocator ILK in config already